starport/templates/ibc/packet.go

```diff
@@ -226,32 +227,21 @@ EventType%[2]vPacket       = "%[3]v_packet"
 			strings.Title(opts.PacketName),
 			opts.PacketName,
 		)
-		content := strings.Replace(f.String(), PlaceholderIBCPacketEvent, replacement, 1)
+		content := replacer.Replace(f.String(), PlaceholderIBCPacketEvent, replacement)
 
 		newFile := genny.NewFileS(path, content)
 		return r.File(newFile)
 	}
 }
 
-func protoTxModify(opts *PacketOptions) genny.RunFn {
+func protoTxModify(replacer placeholder.Replacer, opts *PacketOptions) genny.RunFn {
 	return func(r *genny.Runner) error {
 		path := fmt.Sprintf("proto/%s/tx.proto", opts.ModuleName)
 		f, err := r.Disk.Find(path)
 		if err != nil {
 			return err
 		}
 
-		// Imports
```

Comment:
Hey, was it intentional to rm this one?

Reply:
sorry forgot to mention it in PR description. yes it was intentional, I found a broken placeholder. and after it was fixed it turned out that it is not needed actually (also confirmed this with @lubtd)